systemtest/tests/integration/test.sh

```diff
@@ -33,7 +33,7 @@ if [[ "$ID" == "centos" ]] || [[ "$ID" == "rhel" ]] ; then
 fi
 
 dnf --setopt install_weak_deps=False install -y \
-  yggdrasil podman git-core python3-pip python3-pytest logrotate mosquitto go
+  yggdrasil podman git-core python3-pip python3-pytest logrotate mosquitto go insights-client
```

Comment:
I've added this to the image mode PR, we should merge this after that is so both of these PRs can remain separate.

Reply:
So putting insights-client here will break fedora tests as insights-client doesn't exist (so it fails the git install); we either need to fix that with a smart work around or add --skip-unavailable to the dnf command.